orchestra/admin.py

```diff
@@ -175,10 +175,8 @@ class TimeEntryAdmin(AjaxSelectAdmin):
 
 
 @admin.register(Todo)
-class TodoAdmin(AjaxSelectAdmin):
-    form = make_ajax_form(Todo, {
-        'task': 'tasks',
-    })
+class TodoAdmin(admin.ModelAdmin):
+    autocomplete_fields = ('task', 'parent_todo')
```

Comment:
sanity-checking (no loom video on public repo please :)): does searching for tasks and todos work OK? Meaning are you able to search for the tasks's title/descriptions like you used to, and similarly for the description text of the todo?

Reply:
(discussed offline, works great!)